sirepo/reply.py

```diff
@@ -430,6 +443,14 @@ def _content_type(resp):
             r.set_header("Content-Type", c)
 
         def _cookie(resp):
+            def _delete():
+                for n, p in self._cookies_to_delete:
```

Comment:
I like the model that looks like:

```py
for n, p in self.pkdel('_cookies_to_delete') or ():
```

This ensures the operation is atomic. If it fails, whatever caused it to fail will not cause it to fail again (infinite loop).

Reply:
https://github.com/radiasoft/sirepo/pull/5523/commits/711e56e4b6d7aae82470e59ce8f2c1e71199f412